pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -348,6 +348,47 @@ func sortServicesByCreationTime(services []*model.Service) []*model.Service {
 	return services
 }
 
+func protocolName(p model.Protocol) string {
+	switch plugin.ModelProtocolToListenerProtocol(p) {
+	case plugin.ListenerProtocolHTTP:
+		return "HTTP"
+	case plugin.ListenerProtocolTCP:
+		return "TCP"
+	default:
+		return "UNKNOWN"
+	}
```

Comment:
You shouldn't need this.. i think model.Protocol already has a string function that covers these.

Reply:
I don't see that function defined anywhere